web/templates/move_topic_to_stream.hbs

```diff
@@ -5,7 +5,9 @@
     {{#if only_topic_edit }}
     <p>{{t "Rename topic to:" }}</p>
     {{else}}
-    <p>{{t "Select a channel below or change topic name." }}</p>
+    {{#unless (and (not from_message_actions_popover) (not only_topic_edit))}}
+    <p>{{t "Move messages to." }}</p>
+    {{/unless}}
```

Comment:
How did you determine this is the correct condition to check? 

Also note it's supposed to be a `:` after the `to`.

Reply:
Also should this just be an `if` statement instead of an `unless`? Demorgan's law seems relevant here.

Reply:
> How did you determine this is the correct condition to check?
> 
> Also note it's supposed to be a `:` after the `to`.

in the `move_topic_to_stream.hbs` file the below text in p tag  renders when both `from_message_actions_popover` and `only_topic_edit` are false, and when I looked it up it is basically true when "move topic" modal is open 
![image](https://github.com/zulip/zulip/assets/94672267/4e56be1e-dceb-4c8a-b75b-621d27b538e0)

now as suggested change I had to change the text `Select a channel below or change topic name.`  to `Move messages to: ` in the "move message" modal and had to remove from "move topic" modal. In the previous code given below it was just checking whether it is only "change topic name or not" and if not the later text was rendered in both "move message" and "move topic" modal.
![image](https://github.com/zulip/zulip/assets/94672267/cdf19c0f-818b-4e65-a9fe-7c0894802e7b)

so I introduced a logic when the text will not be rendered when "move topic" modal is opened, to ensure that I introduced negation of logic shown in first picture in this comment, which comes out to be 
`~(p or q) is equivalent to (~p and ~q)`
![image](https://github.com/zulip/zulip/assets/94672267/34e6b15c-f301-4860-912c-22e367f45782)

so as the text under `unless` get's rendered only when the condition under `unless` is false, the message `Move messages to:` does not render when both of them are true, that is when "move topic" modal is opened, but it could happen that in this case the text also might get rendered when `only_edit_topic` is true but since this condition is already checked before coming to this line this does not get rendered.

while `if` can be used to instead of `unless` but then the condition is needs to be changed. Please let me know If I need to change the condition. For now I am fixing the little condition and updating the PR.

Reply:
I think this condition can just be simplified to given the only condition to show this text is when the user opens "Move messages" modal and it is not a topic only edit -
```
{{else if from_message_actions_popover}}
<p>{{t "Move messages to:" }}</p>
{{/if}}
```